code-examples/quarkus-service-registry-quickstart/consumer/src/main/resources/application.properties

```diff
@@ -4,23 +4,23 @@ mp.messaging.incoming.quotes.value.deserializer=io.apicurio.registry.serde.avro.
 mp.messaging.incoming.quotes.apicurio.registry.use-specific-avro-reader=true
 mp.messaging.incoming.quotes.apicurio.registry.avro-datum-provider=io.apicurio.registry.serde.avro.ReflectAvroDatumProvider
 
-%dev.mp.messaging.incoming.quotes.apicurio.auth.service.url=${OAUTH_SERVER_URL:https://identity.api.openshift.com/auth}
-%dev.mp.messaging.incoming.quotes.apicurio.auth.realm=${OAUTH_REALM:rhoas}
-%dev.mp.messaging.incoming.quotes.apicurio.auth.client.id=${CLIENT_ID}
-%dev.mp.messaging.incoming.quotes.apicurio.auth.client.secret=${CLIENT_SECRET}
+%dev.mp.messaging.incoming.quotes.apicurio.auth.service.url=${RHOAS_OAUTH_TOKEN_URL:https://identity.api.openshift.com/auth}
+%dev.mp.messaging.incoming.quotes.apicurio.auth.realm=${RHOAS_OAUTH_REALM:rhoas}
+%dev.mp.messaging.incoming.quotes.apicurio.auth.client.id=${RHOAS_CLIENT_ID}
+%dev.mp.messaging.incoming.quotes.apicurio.auth.client.secret=${RHOAS_CLIENT_SECRET}
 
 mp.messaging.connector.smallrye-kafka.apicurio.registry.url=${REGISTRY_URL}
 
 %test.quarkus.apicurio-registry.devservices.port=8888
 
 ##Kafka servers and auth configuration
 
-%dev.kafka.bootstrap.servers=${BOOTSTRAP_SERVER}
+%dev.kafka.bootstrap.servers=${KAFKA_URL}
```

Comment:
Didn't we changed that to kafka host etc.

Reply:
It hasn't been changed as of now. We needed to pick one of `kafka_bootstrap_server` and `Kafka_host`

Reply:
Kafka host. We need to release new version with it (I have some small changes today to release as well)

Reply:
PR created in [cli#1528](https://github.com/redhat-developer/app-services-cli/pull/1528).
Had missed changing the registry related environment variables in guides, have been updated.